tests/yam/verify_yast2_control_center.pm

```diff
@@ -0,0 +1,25 @@
+# Copyright 2024 SUSE LLC
```

Comment:
I don't see this like a separate test (for now) more like something needed to actually verify the consistency of the self update, cold you please merge it with the test module above?

Reply:
ok, done